### ISSUE TITLE
[Filebeat][MySQL Enterprise] Ingest pipeline parser error for trailing comma

### ISSUE THREAD

**Issue by ebeahan** [elastic/beats#35158]
When used with Elasticsearch 8.7.x, each event processed by the `mysqlenterprise` Filebeat module's ingest pipeline results in an error:

```
Unexpected character (',' (code 44)): expected a value\n at [Source: (org.elasticsearch.common.io.stream.ByteBufferStreamInput); 
```

MySQL JSON-formatted audit logging uses an array with each audit log entry added as a new entry to the array separate by a comma (`,`) character before the newline. In Elasticsearch 8.6 and earlier, the less-restrictive JSON ingest processor didn't fail on the trailing comma. 

Example mysql audit log entries ([source](https://github.com/elastic/beats/blob/main/x-pack/filebeat/module/mysqlenterprise/audit/test/mysql_audit_test.log))

```
{ "timestamp": "2020-10-19 19:21:33", "id": 0, "class": "audit", "event": "startup", "connection_id": 0, "account": { "user": "skip-grants user", "host": "" }, "login": { "user": "", "os": "", "ip": "", "proxy": "" }, "startup_data": { "server_id": 1, "os_version": "x86_64-Linux", "mysql_version": "8.0.22-commercial", "args": ["/usr/local/mysql/bin/mysqld", "--loose-audit-log-format=JSON", "--log-error=log.err", "--pid-file=mysqld.pid", "--port=3306" ] } },
{ "timestamp": "2020-10-19 19:25:51", "id": 0, "class": "connection", "event": "connect", "connection_id": 13, "account": { "user": "root", "host": "localhost" }, "login": { "user": "root", "os": "", "ip": "", "proxy": "" }, "connection_data": { "connection_type": "socket", "status": 0, "db": "", "connection_attributes": { "_pid": "33038", "_platform": "x86_64", "_os": "Linux", "_client_name": "libmysql", "os_user": "root", "_client_version": "8.0.22" } } },
{ "timestamp": "2020-10-19 19:25:51", "id": 1, "class": "general", "event": "status", "connection_id": 13, "account": { "user": "root", "host": "localhost" }, "login": { "user": "root", "os": "", "ip": "", "proxy": "" }, "general_data": { "command": "Query", "sql_command": "select", "query": "select @@version_comment limit 1", "status": 0 } },
```

The error reported here is due to a change in the ingest pipeline JSON processor defaults beginning in Elasticsearch 8.7.0: https://github.com/elastic/elasticsearch/pull/93179.

**Comment by andrewkroh** [comment]
For reference this is a description of the JSON audit log format: https://dev.mysql.com/doc/refman/5.7/en/audit-log-file-formats.html#audit-log-file-json-format

**Comment by efd6** [comment]
Closed by #35160.
